### ISSUE TITLE
settings.js attempts to import schema for system-simple-monitor

### ISSUE THREAD

**Issue by Salohcins** [gnome-hub/gnomehub_extension#34]
**Describe the bug**
Extension settings fail to load, looking for the settings schema for the system-simple-monitor:

```
Error: Schema org.gnome.shell.extensions.simple-system-monitor could not be found for extension gnomehub@gnome-hub.github.io. Please check your installation

Stack trace:
  getSettings@resource:///org/gnome/Shell/Extensions/js/misc/extensionUtils.js:150:15
  Prefs@/home/nick/.local/share/gnome-shell/extensions/gnomehub@gnome-hub.github.io/settings.js:7:41
  @/home/nick/.local/share/gnome-shell/extensions/gnomehub@gnome-hub.github.io/prefs.js:6:23
  _init@resource:///org/gnome/Shell/Extensions/js/extensionsService.js:206:33
  OpenExtensionPrefsAsync/<@resource:///org/gnome/Shell/Extensions/js/extensionsService.js:122:28
  asyncCallback@resource:///org/gnome/gjs/modules/core/overrides/Gio.js:115:22
  run@resource:///org/gnome/Shell/Extensions/js/dbusService.js:177:20
  main@resource:///org/gnome/Shell/Extensions/js/main.js:19:13
  run@resource:///org/gnome/gjs/modules/script/package.js:206:19
  start@resource:///org/gnome/gjs/modules/script/package.js:190:8
  @/usr/share/gnome-shell/org.gnome.Shell.Extensions:1:17
```

**To Reproduce**
1. Go to https://extensions.gnome.org/extension/4913/gnomehub/
2. Click on the settings icon.

**Expected behavior**
Relevant settings screen for the plugin.

**Screenshots**
![image](https://user-images.githubusercontent.com/3887085/160732107-dcea22bf-1e84-4573-8207-cb7fdc6916bf.png)

**Desktop (please complete the following information):**
-  OS: Ubuntu
-  Version: 21.10
-  Gnome Version: 40.5

**Comment by Colton-K** [comment]
Thanks for bringing this up, we will work on fixing it in the next couple days!

**Comment by varora24** [comment]
Issue has been resolved in Pull Request #37 . The update will be made live on the gnome store in a couple of days. Thank you for reporting the bug to us!
